examples/save-load-state/save-load-state.cpp

```diff
@@ -221,7 +221,7 @@ int main(int argc, char ** argv) {
         printf("%s", next_token_str.c_str());
         result2 += next_token_str;
 
-        if (llama_decode(ctx3, llama_batch_get_one(&next_token, 1, n_past, 1))) {
+        if (llama_decode(ctx3, llama_batch_get_one(&next_token, 1))) {
```

Comment:
This will generate a batch for `seq_id == 0` and it needs to be `seq_id == 1`

```bash
make -j && ./llama-save-load-state -m ${some_model}
```

Reply:
Fixed in https://github.com/ggerganov/llama.cpp/pull/9745/commits/6395174a5410ba6634b8256e6cbc4c4625e46907